LocalControl/Control/HoloLensRobot.cs

```diff
@@ -0,0 +1,430 @@
+﻿/*
```

Comment:
This looks like the same file as https://github.com/ooeygui/holobot/blob/master/BigBrain/CS/HoloBot/HoloLensRobot.cs. Can you just link to that file in this project?